config/fastp_config.yaml

```diff
@@ -0,0 +1,22 @@
+project_top_level: /SAN/vyplab/alb_projects/data/4su_better_design_ward_ipsc
+fastp_trimmed_output_folder: 4SU_second_experiment_i3cort/fastp_trimmed/
+merged_fastq_folder: 4SU_second_experiment_i3cort/merged_trimmed/
+sampleCSVpath: /SAN/vyplab/alb_projects/data/4su_better_design_ward_ipsc/foursu_full_experiment.csv
+fastp_path: /SAN/vyplab/alb_projects/tools/fastp
```

Comment:
Again should use fastp installed by conda or a docker container pulled from biocontainers